lesson3/lesson3_3.py

```diff
@@ -0,0 +1,29 @@
+# Найти сумму элементов матрицы
+import random
+
+
+def check_input_digit(string):
+    user_digit = input(string)
+    while not user_digit.isdigit():
+        user_digit = input("Wrong data. Try again: ")
+    return int(user_digit)
+
+
+def generate_matrix(length_input, high_input):
```

Comment:
For the future: name matrix dimensions as
```suggestion
def generate_matrix(matrix_rows_number, matrix_columns_number):
```